src/styles/blocks/_reset.scss

```
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+body {
```

```suggestion
html {
```
its better to place fonts to the `html` tag